cypress/support/commands.ts

```diff
@@ -23,3 +23,13 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add(
+  'visitTargetPage',
```

Comment:
Random idea that came to my mind while reading this: would it be more readable if we refactor this command to 2 different commands:

1. visitTargetPageAsAuthenticatedUser
2. visitTargetPageAsUnauthenticatedUser

:thinking:

Reply:
Yes, I also thought about this 👍 
Will fix